test/adaptors/in-memory-articles.test.ts

```diff
@@ -8,6 +8,9 @@ import NotAnArticle from '../../src/errors/not-an-article';
 import { schema } from '../../src/namespaces';
 import createArticle from '../create-article';
 
+/**
+ * @group unit
```

Comment:
Any way to have a default group, or exclude groups when running? Would rather just tag certain ones.

Reply:
> Any way to have a default group, or exclude groups when running? Would rather just tag certain ones.

I will look into this. It would be an improvement.

Reply:
Opened https://github.com/eugene-manuilov/jest-runner-groups/issues/4#issue-556885899.